cache/remote.go

```diff
@@ -112,9 +112,11 @@ func getAvailableBlobs(ctx context.Context, cs content.Store, chain *solver.Remo
 	if err != nil {
 		return nil, err
 	}
-	var descs []ocispecs.Descriptor
+	var descs = map[digest.Digest]ocispecs.Descriptor{}
```

Comment:
The current patch seems to make output order potentially undeterministic. Could this be a problem?